OrbitGl/ProcessesDataView.cpp

```diff
@@ -158,20 +158,20 @@ void ProcessesDataView::OnSort(int a_Column,
 //-----------------------------------------------------------------------------
 void ProcessesDataView::OnSelect(int a_Index) {
   m_SelectedProcess = GetProcess(a_Index);
+  if (!m_IsRemote) {
+    m_SelectedProcess->ListModules();
+  } else if (m_SelectedProcess->GetModules().size() == 0) {
```

Comment:
I am wondering if we even need this check here. Should this request list of modules on every select, in case they were updated on the server?

Reply:
I see your point, I'd be tempted to leave it as is just to make sure to not introduce an indirect bug.  Also, I think a module load should be detected from the service side perhaps and be automatically sent to the UI instead of the user having to reselect the same process.